binders/difido-testng/src/main/java/il/co/topq/difido/config/AbstractPropertiesConfigFile.java

```diff
@@ -79,22 +54,34 @@ private void initDefaultProperties() {
 			log.warning("Failed to write Difido properties to file");
 		}
 	}
-
-	public String getPropertyAsString(RemoteDifidoOptions option) {
```

Comment:
I think that instead of many methods to convert from String, you can do something like:
public  <T> T getConfig(String key, Class<T> clazz) throws Exception{
        if (configurations.containsKey(key)) {
            String value = configurations.get(key).toString();
            if (clazz.isAssignableFrom(Integer.class) || clazz.isAssignableFrom(int.class)) {
                return (T) Integer.valueOf(value);
            } else if (clazz.isAssignableFrom(Boolean.class) || clazz.isAssignableFrom(boolean.class)) {
                return (T) Boolean.valueOf(value);
            } else if (clazz.isAssignableFrom(Double.class) || clazz.isAssignableFrom(double.class)) {
                return (T) Double.valueOf(value);
            } else return (T) value.toString();
        }
        throw new Exception("Missing configuration key "+ key);
    }

of course, add all items you're supporting.